base_input_topic: "/front_camera/image_raw"
face_detected_image_topic: "/face_detector/raw_image"
face_tracking: 1
display_original_image: 0
display_tracking_image: 1
haar_file_face: /home/yago/workspaces/catkin_ws/src/eyecu/eyecu/data/face.xml